contrib/drivers/ceph/ceph_metrics.go

```diff
@@ -138,27 +127,33 @@ func (d *MetricDriver) CollectMetrics(metricsList []string, instanceID string) (
 
 		metric := &model.MetricSpec{
 			InstanceID:   instanceID,
-			InstanceName: "",
-			Job:          "OpenSDS",
+			InstanceName: "opensds_ceph_metrics",
```

Comment:
Why do we hard code this? if there is no  different name for the instance , we can keep instanceID as name?

Reply:
@NajmudheenCT Thanks for reviewing. I am using instanceID as InstanceName as you suggested.
Thanks.